README.md

```diff
@@ -81,6 +81,25 @@ WebPack will search your `node_modules` folder by default, so if you include `ab
 
 If that doesn't work for some reason (e.g. you are using a custom webpack target), you can reference the `ably.js` static file directly: `require('ably/build/ably.js');` (or `import * as Ably from 'ably/build/ably.js'` for typescript / ES6 modules).
 
+#### Modular (tree-shakable) variant
+
+Aimed at those who are concerned about their app’s bundle size, the modular variant of the library allows you to create a client which has only the functionality that you choose. Unused functionality can then be tree-shaken by your module bundler.
```

Comment:
I do think we should emphasise here that the API is the same as for the default clients, and be explicit that you can develop an application with a default client and then switch over to the modular client only when you want to start using it in production.

Reply:
I also think it's worth being clear here exactly what's required for a working realtime client, ie at least one transport and at least one http client

Reply:
> I do think we should emphasise here that the API is the same as for the default clients, and be explicit that you can develop an application with a default client and then switch over to the modular client only when you want to start using it in production.

Done.

> I also think it's worth being clear here exactly what's required for a working realtime client, ie at least one transport and at least one http client

I already mentioned this in the typings (and hence the TypeDoc docs), but have added it to the README too.